src/vs/workbench/browser/parts/editor/tabsTitleControl.ts

```diff
@@ -211,6 +214,10 @@ export class TabsTitleControl extends TitleControl {
 			}
 		}));
 
+		// Configuration updates
+		this.toUnbind.push(this.configurationService.onDidUpdateConfiguration(() => this.onConfigurationChanged()));
+		this.onConfigurationChanged();
```

Comment:
This call seems to cause an update even though we are in the middle of creating the control. I would add an extra flag to not cause an update if this method is being called from construction phase.

Reply:
fixed in d925602